BoxContentSDK/BoxContentSDK/OAuth2/BOXParallelOAuth2Session.m

```diff
@@ -46,8 +46,8 @@ - (void)performRefreshTokenGrant:(NSString *)expiredAccessToken withCompletionBl
 }
 
 - (void)performRefreshTokenGrant:(NSString *)expiredAccessToken
-      newAccessTokenExpirationAt:(NSString *)accessTokenExpirationTimestamp
-     newRefreshTokenExpirationAt:(NSString *)refreshTokenExpirationTimestamp
+      newAccessTokenExpirationAt:(NSNumber *)accessTokenExpirationTimestamp
```

Comment:
should we clean up the naming as well? An nsnumber is likely not a timestamp, probably an offset?